core/ipc/twin/common/src/main/java/org/opennms/core/ipc/twin/common/AbstractTwinPublisher.java

```diff
@@ -243,7 +257,12 @@ public void publish(T obj) throws IOException {
                     span.setTag(TAG_VERSION, twinUpdate.getVersion());
                     span.setTag(TAG_SESSION_ID, twinUpdate.getSessionId());
                     handleSinkUpdate(twinUpdate);
+                    String sinkUpdateMetricName = sessionKey.location != null ?
+                            MetricRegistry.name(sessionKey.location, sessionKey.key, SINK_UPDATE_SENT) :
+                            MetricRegistry.name(sessionKey.key, SINK_UPDATE_SENT);
```

Comment:
As these names for a tree, maybe we should use the same number of elements in the name. is using `*` allowed?

Reply:
> As these names for a tree, maybe we should use the same number of elements in the name. is using `*` allowed?

mbean naming should take care of it.

![Screen Shot 2021-12-15 at 4 55 28 PM](https://user-images.githubusercontent.com/17052395/146271612-d7a523cd-97ac-489a-8643-982a9ec574bf.png)
.